log/logger.go

```diff
@@ -127,13 +127,13 @@ type logger struct {
 	h   *swapHandler
 }
 
-func (l *logger) write(msg string, lvl Lvl, ctx []interface{}) {
+func (l *logger) write(msg string, lvl Lvl, ctx []interface{}, skip int) {
 	l.h.Log(&Record{
 		Time: time.Now(),
 		Lvl:  lvl,
 		Msg:  msg,
 		Ctx:  newContext(l.ctx, ctx),
-		Call: stack.Caller(2),
+		Call: stack.Caller(skip),
```

Comment:
@nonsense why was this change introduced? i mean, not better to have this adjustable from one spot instead of 12 others?

Reply:
This is needed, so that we have the `Output` method, which takes a different `skip` level.

Reply:
ok, but maybe we could have a `const` for the default value of `2`?

Reply:
why do we need this change?

Reply:
@zelig it is explained above why we need it - See function `Respond` and the `Output` method. This helps us track the caller line more stacks up, than just 2.